### PR TITLE
Document the bin/fixmystreet.com/generate_council_location script

### PR DESCRIPTION
**Pull request by chrismytton**
Add some documentation to the `bin/fixmystreet.com/generate_council_location` script.

Also print out usage if no arguments are provided.

### CODE REVIEW

File: bin/fixmystreet.com/generate_council_location [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+
 use strict;
 use JSON;
 use LWP::Simple;
 
 my $id = shift;
 
+die "Usage: $0 <mapit_area_id>\n" unless $id;
+
 my $url = sprintf( 'http://mapit.mysociety.org/area/%s/geometry', $id);
```
Comment (dracos) [inline bin/fixmystreet.com/generate_council_location:22]: While we're here, switching to https would speed it up a tiny bit ;)

**Review by dracos — CHANGES_REQUESTED** [review]
tiny thing :)

File: bin/fixmystreet.com/generate_council_location (hunk 1 above)
Comment (chrismytton) [reply bin/fixmystreet.com/generate_council_location]: Done in 40655d7d4d744aaf2bf9a1fe40573cc5af5ea567

**Review by dracos — APPROVED** [review]
Fine to squash together, I'd say :)

**Pull request merged**